postprocessing_variant_calls/maf/tag/tag_process.py

```diff
@@ -268,5 +269,124 @@ def traceback(
     return 0
 
 
+@app.command(
+    "by_rules",
+    help="Tag a variant in a MAF file based on criterion provided by input rules JSON file",
+)
+def by_maf(
+    maf: Path = typer.Option(
+        ...,
+        "--maf",
+        "-m",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        writable=False,
+        readable=True,
+        resolve_path=True,
+        help="MAF file to tag",
+    ),
+    rules: Path = typer.Option(
+        ...,
+        "--rules",
+        "-r",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        writable=False,
+        readable=True,
+        resolve_path=True,
+        help="Intervals JSON file containing criterion to tag input MAF by",
+    ),
+    output_maf: Path = typer.Option(
+        "output.maf", "--output", "-o", help="Maf output file name."
+    ),
+    separator: str = typer.Option(
+        "tsv",
+        "--separator",
+        "-sep",
+        help="Specify a separator for delimited data.",
+        callback=check_separator,
+    ),
+):
+    # prep maf
+    mafa = MAFFile(maf, separator)
+    typer.secho(
+        f"Tagging Maf with criteria from input rules JSON file",
+        fg=typer.colors.BRIGHT_GREEN,
+    )
+    rules_file = RulesFile(rules)
+    tagged_by_rules_maf = mafa.tag_by_rules(rules_file.data_frame)
+    
+    typer.secho(f"Writing Delimited file: {output_maf}", fg=typer.colors.BRIGHT_GREEN)
+    tagged_by_rules_maf.to_csv(f"{output_maf}".format(outputFile=output_maf), index=False, sep="\t")
+    return 0
+
+
+@app.command(
+    "by_access",
+    help="Tag a variant in a MAF file based on criterion stated by the SNV/indels ACCESS pipeline workflow",
+)
+def by_access(
```

Comment:
Is this different from `by_rules`? 

Could we just call this `access` instead of `by_access`?

Reply:
@buehlere Yes, by_access is the function that combines by_rules (now renamed to by_variant annotations), tag hotspots, and tag germline. I'll rename it to access now!